src/drivers/intel/cavs/Kconfig

```diff
@@ -22,6 +22,7 @@ config CAVS_DMIC_FIR_FULL
 	select CAVS_DMIC_FIR_DECIMATE_BY_5
```

Comment:
@singalsuo In commit message "adds to driver capability to" is not right. Did you mean "adds driver capability to capture" ? Maybe even just "adds capability to capture" as the patch is anyways to dmic driver code and you have "dmic:" in the summary line.

Reply:
Thanks, it's fixed now.